### PR TITLE
Complete Database Scripts

### PR DESCRIPTION
**Pull request by UNDogMan**

### CODE REVIEW

File: DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿CREATE PROCEDURE [dbo].[DeleteTask]
+	@TaskId int
+AS
+	DELETE [Tasks] WHERE TaskId = @TaskId;
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql:4]: I forgot <code>FROM</code>

File: DIMS-Core.Database/DIMS-Core.Database/Tables/TaskTracks.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[TaskTracks]
+(
+	[TaskTrackId] INT NOT NULL IDENTITY (1, 1),
+	[UserTaskId] INT NOT NULL,
+	[TrackDate] DATE NOT NULL,
+	[TrackNote] NVARCHAR(250) NOT NULL,
+
+	CONSTRAINT PK_TaskTracks_TaskTrackId PRIMARY KEY(TaskTrackId),
+	FOREIGN KEY (UserTaskId) REFERENCES UserTasks(UserTaskId)
```
Comment (PushMeNow) [inline DIMS-Core.Database/DIMS-Core.Database/Tables/TaskTracks.sql:9]: Write name for your FK and on delete/update actions

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi dude!
I left some comments. Let me know if any questions

File: DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql (hunk 1 above)
Comment (UNDogMan) [reply DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteTask.sql:4]: Why we need `FROM` here?

**Review by PushMeNow — APPROVED** [review]
Approve

**Pull request merged**